translations/ui-orders/en.json

```diff
@@ -501,6 +501,8 @@
   "receiving.cancelBtn": "Cancel",
   "receiving.dateOrdered": "Date Ordered",
   "receiving.format": "Format",
+  "receiving.itemStatus.inProcess": "In process",
+  "receiving.itemStatus.onOrder": "On order",
   "receiving.itemStatus.received": "Received",
```

Comment:
I think "Received" can be removed as an itemStatus.  It has been replaced by "In process"

Reply:
ok